lark-stubs/load_grammar.pyi

```diff
@@ -24,3 +25,6 @@ class GrammarBuilder:
     def validate(self) -> None: ...
 
     def build(self) -> Grammar: ...
+
+
+def find_grammar_errors(text: str, start: str='start') -> List[UnexpectedInput, str]: ...
```

Comment:
List[Tuple[]]

Reply:
Ach, my bad. Fixed.

Reply:
I recognize this instantly because I also do this regularly.